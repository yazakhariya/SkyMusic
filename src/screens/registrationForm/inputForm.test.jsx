import InputForm from './inputForm';
import { screen, render } from '@testing-library/react';

describe("<InputForm />", () => { 
    describe("Snapshots tests", () => { 
        it("should render successfully and match snapshot", () => { 
            const { container } = render(<InputForm />); 
            expect(container).toMatchSnapshot(); 
        }); 
    });

    describe("Attributes tests", () => { 
        it("should set type attribute", () => { 
            const type = "text"; 
            render(<InputForm type={type} />); 
            expect(screen.getByPlaceholderText("Почта")).toHaveAttribute("type", type); 
        });
    });
});