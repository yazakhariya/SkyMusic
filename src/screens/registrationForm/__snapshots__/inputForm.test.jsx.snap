// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputForm /> Snapshots tests should render successfully and match snapshot 1`] = `
<div>
  <input
    required=""
    value=""
  />
</div>
`;
